hack/istio/install-runtimes-demo.sh

```diff
@@ -51,14 +53,24 @@ install_runtimes_demo() {
     $CLIENT_EXE adm policy add-scc-to-user anyuid -z default -n ${RUNTIMES}
   fi
 
-  ${CLIENT_EXE} label namespace ${RUNTIMES} istio-injection=enabled --overwrite=true
+  if [ "${AUTO_INJECTION}" != "" ]; then
```

Comment:
Shouldn't this be explicitly checking for true - first, to be consistent with all the other ones you added in the other scripts, and second, because this will inject sidecars even if someone sets the value to `"false"`:
```suggestion
  if [ "${AUTO_INJECTION}" == "true" ]; then
```